addons/mail/static/src/messaging/component/chatter/chatter.scss

```diff
@@ -9,6 +9,11 @@
     overflow: auto;
 }
 
+.o_Chatter_composer {
+    border-right: 1px solid gray('300');
```

Comment:
The whole thread need this border I think (on the right and also bottom) so to include the message area, not just the composer.

But also I'm not exactly sure if we also want that border when the chatter is on the right of the screen, maybe not?

Reply:
No border found around thread in enterprise version, the border on the bottom is the border of the last message. And based on community version in master, there should be no left border instead of having right and bottom order. So I'll make sure there is not left border when chatter is